### ISSUE TITLE
word-form analysis presentation (includes orthographical normalization)

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/itwewina#13]
In the analysis column on the right on both the landing and follow-up pages, the underlying concepts have been the following:

1. Present the normative form of the search string, if it can be analyzed as a Cree word with spelling-relaxation [Done] (#12)

2. Present first the stem/root (not the lemma) that is the <stem>…</stem> element in the XML source [Done]

3. Then present all the features in their order of occurrence. The lemma/stem has not been made part of this (as it is already presented with the √ symbol. [Done] (#12)

4. Relabeling from itwewina.relabel are applied to the underlying FST analysis tags [Done]

Examples

a. e-wapamat
a1. Normalization: ê-wâpamât
a2. Analysis: √wâpam + Preverb: ê + Verb + Transitive Animate + Conjunct + Tense: Present + Actor: a3. 2nd person singular + Goal: 3rd person singular (<- PV/e V TA Cnj Prs 2Sg 3SgO)

b. ma-mah-miyo-na-nah-nipaw
b1. Normalization: ma-mâh-miyo-na-nâh-nipâw
b2. Analysis: √nipâ- + Reduplication: Weak + Reduplication: Strong + Preverb: miyo + Reduplication: Weak + Reduplication: Strong + Verb + Animate Intransitive + Independent + Tense: Present + Actor: 3rd person singular (<- RdplW+RdplS+PVmiyo+RdplW+RdplS+nipâw+V+AI+Ind+Prs+3Sg)

c. niki-nitawi-kiskinwahamakosin
c1. Normalization: nikî-nitawi-kiskinwahamâkosin
c2. Analysis: √kiskinwahamâkosi- + Preverb: nitawi + Verb + Animate Intransitive + Independent + Tense: Past+ Actor: 1st person singular ( PV/nitawi+kiskinwahamâkosiw+V+AI+Ind+Prt+1Sg)

REGULAR EXPRESSION DEFINING PREFIXES

* ( (RdplW\+)? (RdplS\+ )? PV/.+ )* ( RdplW\+)? (RdplS\+ )? [lemma] \+V \+ …
* Note that reduplication does not apply for grammatical preverbs. These are some 10-20, e.g. ê-, ka-, kâ-, but this is ruled out in the LEXC code by having a separate contlex for grammatical preverbs, before the lexical ones that can have reduplication

Possible reduplication + proverb combinatorics (not all):

nipâw
na-nipâw
nâh-nipâw
na-nâh-nipâw
miyo-nipâw
ma-miyo-nipâw
mâh-miyo-nipâw
ma-mâh-miyo-nipâw
ma-miyo-na-nipâw
mâh-miyo-na-nipâw
ma-miyo-nâh-nipâw
mâh-miyo-nâh-nipâw
ma-miyo-na-nâh-nipâw
mâh-miyo-na-nâh-nipâw
ma-mâh-miyo-na-nâh-nipâw
mâh-miyo-nipâw
ma-mâh-miyo-nipâw
…

**Comment by aarppe** [comment]
Issue #12 is the first presentation element (point 1, examples a1, b1, c1 above) of this issue (#13)

**Comment by aarppe** [comment]
Results of search (analyses and presented normative forms) are not the same when using SRO circumflex vs. SRO macron. See search with 'nipihk' below:

![image](https://user-images.githubusercontent.com/15003407/44677122-6666e800-a9f2-11e8-8be9-523993ec16a0.png)

![image](https://user-images.githubusercontent.com/15003407/44677175-84344d00-a9f2-11e8-8a17-580bc03806bb.png)

Besides fixing this, should we also have regression tests for this, i.e. the search results in all three orthographies?

**Comment by eddieantonio** [comment]
https://github.com/UAlbertaALTLab/itwewina/issues/13#issuecomment-416317307 appears to be a completely unrelated issue. Could you make a new issue report for it?

**Comment by eddieantonio** [comment]
Assigning this to @aarppe, since we need ~~[configs/language_specific_rules/tagsets/crk.tagset](https://github.com/UAlbertaALTLab/itwewina/blob/development/neahtta/configs/language_specific_rules/tagsets/crk.tagset)~~ to be updated.

**Comment by aarppe** [comment]
Will take a look. Interestingly, I have never earlier touched that particular file (for any language) within the NDS directory/file labyrinth.

**Comment by eddieantonio** [comment]
Passing you the nakki to update [configs/language_specific_rules/user_friendly_tags/itwewina.relabel](https://github.com/UAlbertaALTLab/itwewina/blob/development/neahtta/configs/language_specific_rules/user_friendly_tags/itwewina.relabel)

**Comment by eddieantonio** [comment]
Condition to resolve this issue: Waiting on #36 to wrap up.

**Comment by aarppe** [comment]
Issue #36 should have been done now, with the above-mentioned file checked-in.

**Comment by eddieantonio** [comment]
This one is done!
